modules/user-access/pages/vlac.adoc

```diff
@@ -42,7 +42,7 @@ Below is the basic workflow on using the VLAC to control data access for a datab
 
 There are three main options for tagging vertices.
 
-. <<_add_tags_on_existing_data>>: A user with the privilege `ACCESS_TAG` and `WRITE_DATA` privileges on the base graph can create and run a DML query that sets tags on selected individual vertices.
+. <<_add_tags_on_existing_data>>: A user with the privilege `ACCESS_TAG` and `UPDATE_DATA` privileges on the base graph can create and run a DML query that sets tags on selected individual vertices.
```

Comment:
The `UPDATE_DATA` can actually be removed, because read/update tags only require `ACCESS_TAG` privilege.

Reply:
```suggestion
. <<_add_tags_on_existing_data>>: A user with the privilege `ACCESS_TAG` privileges on the base graph can create and run a DML query that sets tags on selected individual vertices.
```